web_ui/src/shared/components/label-tree-view/label-tree-view-item/group-edition-mode/group-creation.component.tsx

```diff
@@ -101,7 +110,8 @@ export const GroupCreation = ({
                     <Flex gap={'size-100'} width={'100%'} alignItems={'center'}>
                         <Flex width={'100%'}>
                             <TooltipTrigger placement={'top'}>
-                                <LimitedTextField
+                                <TextField
+                                    maxLength={100}
```

Comment:
```suggestion
                                    maxLength={MAX_NUMBER_OF_CHARACTERS_OF_GROUP}
```

Or feel free to use any other descriptive name that fits better

Reply:
ok, i shall update it on my next PR. ty!